src/lib/fauna-command.js

```diff
@@ -85,9 +84,7 @@ class FaunaCommand extends Command {
   mapConnectionError({ err, connectionOptions }) {
     if (err instanceof errors.Unauthorized) {
       return this.error(
-        `Could not Connect to ${stringifyEndpoint(
-          connectionOptions
-        )} Unauthorized Secret`
+        `Could not Connect to ${connectionOptions.url} Unauthorized Secret`
```

Comment:
```suggestion
        `Could not connect to ${connectionOptions.url}: unauthorized secret`
```

Reply:
hmm yeah I'm not sure that we want to do that, my bias would be to just leave it out